pkgs/development/libraries/wolfssl/default.nix

```diff
@@ -6,73 +6,102 @@
 , util-linux
 , openssl
 }:
-
-stdenv.mkDerivation rec {
-  pname = "wolfssl";
-  version = "5.6.3";
-
-  src = fetchFromGitHub {
+let
+  defaultVersion = "5.6.3";
+  defaultSrc = fetchFromGitHub {
     owner = "wolfSSL";
     repo = "wolfssl";
-    rev = "refs/tags/v${version}-stable";
+    rev = "refs/tags/v${defaultVersion}-stable";
     hash = "sha256-UN4zs+Rxh/bsLD1BQA+f1YN/UOJ6OB2HduhoetEp10Y=";
   };
 
-  postPatch = ''
-    patchShebangs ./scripts
-    # ocsp tests require network access
-    sed -i -e '/ocsp\.test/d' -e '/ocsp-stapling\.test/d' scripts/include.am
-    # ensure test detects musl-based systems too
-    substituteInPlace scripts/ocsp-stapling2.test \
-      --replace '"linux-gnu"' '"linux-"'
-  '';
+  buildWolfSSL =
```

Comment:
Why do we need this helper? We only have one version and people can use overrideAttrs.